### PR TITLE
Add test coverage for main route

### PR DESCRIPTION
**Pull request by maxalmonte14**
Hello 👋

This PR solves https://github.com/reliqarts/creoletranslator/issues/1

Thanks in advance for taking the time to review it.

### CODE REVIEW

File: tests/Feature/HomeTest.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+
+final class HomeTest extends TestCase
+{
+    /** @test */
+    public function it_returns_ok_status_code()
```
Comment (reliq) [inline tests/Feature/HomeTest.php]:
```suggestion
    public function testIndexIsOK(): void
```

File: tests/Feature/HomeTest.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+
+final class HomeTest extends TestCase
+{
+    /** @test */
```
Comment (reliq) [inline tests/Feature/HomeTest.php]:
```suggestion
```

File: .travis.yml [hunk 3]
```diff
@@ -16,6 +16,9 @@ cache:
 before_install:
   - travis_retry composer self-update
 
+before_script:
+  - cp .env.example .env
+
```
Comment (reliq) [inline .travis.yml]: This should fix the failing test:

```suggestion
- php artisan key:generate

```

---
Please also move this `before_script` section to just before the `script` section in the file.

**Comment by maxalmonte14** [comment]
I think it's needed to run `npm install` after generating the key with `php artisan generate:key` in order to this work.

**Comment by maxalmonte14** [comment]
I mean, React must be installed and running for this to work.

File: .travis.yml [hunk 4]
```diff
@@ -19,6 +19,10 @@ before_install:
 install:
   - travis_retry composer install --no-interaction --prefer-dist --no-suggest
 
+before_script:
+  - cp .env.example .env
+  - php artisan key:generate
+
```
Comment (reliq) [inline .travis.yml]:
```suggestion
- php artisan config:clear
- php artisan serve &

```

This may be due to the server not running.

**Comment by reliq** [comment]
I have an idea. Don't try npm yet.

> I mean, React must be installed and running for this to work.

Probably not, the server just needs to respond positively in this case. Let's try that last suggestion.

**Comment by maxalmonte14** [comment]
> I have an idea. Don't try npm yet.
> 
> > I mean, React must be installed and running for this to work.
> 
> Probably not, the server just needs to respond positively in this case. Let's try that last suggestion.

It keeps failing in my local machine, but after a `npm run dev` it runs perfectly.

**Comment by reliq** [comment]
> 
> It keeps failing in my local machine, but after a `npm run dev` it runs perfectly.

OK. Let's try it. 

Add the to `cache/directories` section:
```
- node_modules
```

to `before_install`:
```
- nvm install 'lts/*'
- npm i -g npm
```

Add the following to `install` section:
```
 - npm install --no-audit --no-progress --quiet
 - npm run production
```

**Pull request merged**